authors/apps/articles/tests/test_search.py

```diff
@@ -0,0 +1,34 @@
+from django.urls import reverse
+
+from rest_framework import status
+
+from authors.apps.articles.tests.base_class import ArticlesBaseTest
+from .test_data import valid_article
+from ..models import Article
+
+
+class TestSearchArticle(ArticlesBaseTest):
+
+    def test_search_by_keyword(self):
+        """
+        Tests if a user can search articles by keyword
+        """
+        self.add_article()
+        response = self.client.get(self.articles_url+"?keyword=the")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Also please add as to your reviewer's list there are only two people there

Reply:
@wasibani-roy, Thank you for reminding me but i have added them now